logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/client/ClearClientDataUseCase.kt

```diff
@@ -39,19 +42,22 @@ interface ClearClientDataUseCase {
 
 internal class ClearClientDataUseCaseImpl internal constructor(
     private val mlsClientProvider: MLSClientProvider,
-    private val proteusClientProvider: ProteusClientProvider
+    private val proteusClientProvider: ProteusClientProvider,
+    private val dispatchers: KaliumDispatcher = KaliumDispatcherImpl
 ) : ClearClientDataUseCase {
 
     override suspend operator fun invoke() {
-        clearCrypto()
-            .onSuccess { success ->
-                if (!success) {
-                    kaliumLogger.e("Did not clear crypto storage")
+        withContext(dispatchers.io) {
+            clearCrypto()
```

Comment:
we can add withContext to clearCrypto() function directly, it will be better since it will guaranty main safety to any consumer

Reply:
Yeah it's added here #1428. We missed to merge this PR into develop as well